include/itkTileMontage.hxx

```diff
@@ -280,11 +282,24 @@ TileMontage<TImageType, TCoordinate>::RegisterPair(TileIndexType fixed, TileInde
 
   m_CandidateConfidences[regLinearIndex] = m_PCM->GetConfidences();
   m_TransformCandidates[regLinearIndex].resize(offsets.size());
-  PointType p0;
-  p0.Fill(0.0);
   for (unsigned i = 0; i < offsets.size(); i++)
   {
-    m_TransformCandidates[regLinearIndex][i] = offsets[i] - p0;
+    if (m_CandidateConfidences[regLinearIndex][i] == 0)
+    {
+      m_CandidateConfidences[regLinearIndex][i] = std::numeric_limits<TCoordinate>::epsilon();
+    }
```

Comment:
I'm not sure I understand how/why a confidence of 0 was previously interpreted as NaN?

Reply:
Dividing by zero produces a `NaN`:
https://github.com/InsightSoftwareConsortium/ITKMontage/blob/33b6b742dc285e35e5d44b9e9e2c2eab1a213a2a/include/itkTileMontage.hxx#L540